### PR TITLE
Update fabric to 1.10.2 stable

### PR DESCRIPTION
**Pull request by jcftang**
This brings the fabric package up to date

### CODE REVIEW

File: pkgs/fabric.yaml [hunk 1]
```diff
@@ -4,5 +4,16 @@ dependencies:
...
-    key: tar.gz:ab3mo2v654775els4euq43sipyr2oiu3
+- key: tar.gz:mzvd3kjconjs2osj3unmkpr5oiof5qxs
+  url: https://pypi.python.org/packages/source/F/Fabric/Fabric-1.10.2.tar.gz
+
+build_stages:
+
+- name: install
+  after: setup_dirs
+  handler: bash
+  bash: |
```
Comment (ahmadia) [inline pkgs/fabric.yaml]: should this be a setuptools or a distutils install?

Comment (jcftang) [inline pkgs/fabric.yaml]: It was failing to create the profile due to a write permission error, the build completes if I build/install manually in this profile.

Comment (ahmadia) [inline pkgs/fabric.yaml]: huh, weird.  maybe put a comment in the file explaining what's going on for the next person who looks at this.

Comment (jcftang) [inline pkgs/fabric.yaml]: Done, I've amended the commit

**Comment by jcftang** [comment]
thanks!

File: pkgs/fabric.yaml (hunk 1 above)
Comment (certik) [inline pkgs/fabric.yaml]: What *exactly* is the problem here? This seems to be fixing the symptoms, without understanding the root of the cause.

Comment (jcftang) [inline pkgs/fabric.yaml]: This was the error at profile generation...

[profile|ERROR] hit command failed: [Errno 13] [Errno 13] Permission denied: '/Users/jtang/.hashdist/bld/profile/t63wczbqichu/lib/python2.7/site-packages/tests/etree13' in silent_makedirs(u'/Users/jtang/.hashdist/bld/profile/t63wczbqichu/lib/python2.7/site-packages/tests/etree13',)

Comment (certik) [inline pkgs/fabric.yaml]: So is the problem that the package is installing a `tests` directory directly into the Python's package `site-packages`? And why is this avoided by your patch? What is different?

Comment (jcftang) [inline pkgs/fabric.yaml]: To be honest, I didn't spend much time investigating, it was what worked for me at the time. I think this has arisen a few times for me in the past, I previously just swapped out setuptools for distutils or vice versa and the issue goes away.

**Pull request merged**